docs/api/topics/firefox_os_addons.rst

```diff
@@ -98,6 +100,54 @@ Add-on Search
     :status 200: successfully completed.
 
 
+Add-on Versions
+===============
+
+.. http:get:: /api/v2/extensions/extension/(int:id)|(string:slug)/versions/
+
+    .. note:: Non public add-ons versions can only be viewed by their authors.
+
+    Returns a list of versions attached to an add-on.
+
+    **Request**
+
+    The standard :ref:`list-query-params-label`.
+
+    **Response**
+
+    :param meta: :ref:`meta-response-label`.
+    :type meta: object
+    :param objects: A :ref:`listing <objects-response-label>` of :ref:`add-on versions <addon-version-response-label>`.
+    :type objects: array
+
+    :status 200: successfully completed.
+    :status 403: not allowed.
+    :status 404: add-on not found.
+
+.. _addon-version-response-label:
```

Comment:
Can we add an example representation of this? We refer to this label all over the place, it'd be nice to see what it actually looks like.